packages/runtime/datastore/src/channelStorageService.ts

```diff
@@ -48,7 +50,10 @@ export class ChannelStorageService implements IChannelStorageService {
         if (blob !== undefined) {
             return blob;
         }
-        return this.storage.readBlob(id);
+        const blobP = this.storage.readBlob(id);
+        blobP.catch((error) => this.logger.sendErrorEvent({ eventName: "readBlobError"}, error));
```

Comment:
It would be great for name of even reflect what type of code issues it. I.e. maybe ChannelStorageBlobError, or something like that?
For context - we expect no failures here to ever happen, vs. the code in BlobManager - this one will hit errors because lifetime of those blobs is less well defined.

Reply:
That said, if all code paths add some reasonable prefix to events (like you have below with "RemoteChannelContext"), then shorter name is likely fine